next.config.js

```diff
@@ -6,4 +12,7 @@ module.exports = {
     locales: ['en-CA'],
     defaultLocale: 'en-CA',
   },
+  env: {
+    BASE_URL: baseUrl,
+  },
```

Comment:
Now `process.env.BASE_URL` is usable everywhere and should always "work", without the need for an env var being set locally.